Improve Lisk Service setup guide - Closes #1446 (#1452)
LiskHQ/lisk-docs: modules/ROOT/pages/setup/docker.adoc

== Configuring Lisk Core

Use the Lisk Core CLI to start Lisk Core with the desired configurations.

* The `--network` flag determines the blockchain network Lisk Core connects to.

> * `--api-ws` enables the WebSocket API of the node, if not already enabled in the config.json file.

Reviewer: ```suggestion
* `--api-ws` enables the WebSocket API of the node, if not already enabled within the config.json file.
```

* `--api-ws-host=0.0.0.0` allows remote servers to connect to the Lisk Core WS API.
If this option is not set, it defaults to `127.0.0.1`.
* `--api-ws-port=8080` the port for the WebSocket API.

[source,bash]
----
lisk-core start --network mainnet --api-ws --api-ws-host=0.0.0.0 --api-ws-port=8080
----

> To connect to a different network, replace `mainnet` with either `testnet` or `devnet`.

Reviewer: ```suggestion
To connect to a different network, replace `mainnet` with `testnet`, `alphanet` or `betanet`. To start your own private network for development purposes use `devnet`.
```

Author: do we want to publicly document alphanet and betanet?

Reviewer: We can leave it as is if we do not specify them in any other product documentation. Please feel free to resolve the comment.

Author: ```suggestion
To connect to a different network, replace `mainnet` with either `testnet` or, to start your own private network for development purposes use `devnet`.
```
